navigator_engine/pluggable_logic/conditional_functions.py

```diff
@@ -81,9 +81,34 @@ def check_spectrum_file(indicators: list[str], engine: DecisionEngine) -> bool:
                 f'Indicator "{indicator}" not found in Spectrum checklist'
             )
     indicator_checks = checklist[checklist['ID'].isin(indicators)]
-    indicator_checks['Status'] = indicator_checks['Status'].replace(
-        [0, 'FALSE', 'F', 'false', 'f'],
-        value=False,
-    )
+    indicator_checks['Status'].replace([0, 'FALSE', 'F', 'false', 'f'],
+                                       value=False,
+                                       inplace=True)
+
```

Comment:
I think your branch has been based of an older version of master, or something.  I altered these lines to avoid using the "inplace" param: https://towardsdatascience.com/why-you-should-probably-never-use-pandas-inplace-true-9f9f211849e4